cinema/urls.py

```diff
@@ -1 +1,117 @@
-# write urls here
+from django.urls import path
+from rest_framework import routers
+
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+urlpatterns = [
```

Comment:
There is a way to add a single `path` to the `urlpatterns` using the `router` created above.
Right now it's not even used at all.